frontend/src/pages/Home.tsx

```diff
@@ -0,0 +1,188 @@
+            State of California.
+          </p>
+        </li>
+        <li>
+          <p className={styles.item}>
+            Upon qualification, the Certified Interior Designer will be identified by an individual
+            number on a certificate, a stamp to identify interior design nonstructural and
+            non-seismic drawings and documents, and an identification card (ID).
+          </p>
+        </li>
+        <li>
+          <p className={styles.item}>
+            Certification in California is a Title Act, not a license or registration (Practice
+            Act). The Certified Interior Designer’s Title Act is the only thing the state
+            recognizes.
+          </p>
+        </li>
+      </ul>
+      <button className={styles.apply} type="button">
+        Apply Now
+      </button>
+      <div className={styles.textDivider2}>
+        <div className={styles.heading}>Pathways to Certification</div>
+      </div>
+
+      <p>
+        To become a CID, you must:
+        <ul className={styles.list}>
+          <li>
+            Have a minimum of 6 years of combined accredited interior design education and
+            experience <strong>OR</strong> a minimum of 8 years of combined interior design
+            education and experience.
+          </li>
+          <li>
+            Provide evidence of competing the required education and/or diversified design work
+            experience in interior design, architecture, or building design.
+          </li>
+          <li>Pass the IDEX® California Exam.</li>
+        </ul>
+        There are four paths to qualification, candidates must meet one of the following path
+        criteria and each is listed with proper qualifications below:
+      </p>
+
+      <div className={styles.pathwaysContainer}>
+        <div className={styles.pathwayRows}>
+          <div className={styles.pathwayContainer}>
+            <h2 className={styles.pathName}>Path 1</h2>
+            <div className={styles.pathwayText}>
+              <span className={styles.red}>Qualifications:</span>
+              Candidates who meet the education requirement, and have <strong>NOT</strong> met the
+              work experience requirement, may opt first to apply to take the IDEX® California
+              Exam. Candidates must meet one of these qualifications to sit for the IDEX®
+              California Exam:
+              <ol className={styles.olist}>
+                <li>
+                  At least 40 semester core units from an accredited design program (60 quarter
+                  units)
+                </li>
+                <li>5-8 years of diversified design experience</li>
+              </ol>
+            </div>
+          </div>
+          <div className={styles.pathwayContainer}>
+            <h2 className={styles.pathName}>Path 2</h2>
+            <div className={styles.pathwayText}>
+              <span className={styles.red}>Qualifications: </span>
```

Comment:
Lets make the lines 88-91 into a compoenent called pathwayItem for example and then when on the home page we use it like 
`<pathwayItem path = "1">
     Candidates ......
</pathwayItem>
`

This way if any thing in the future changes about the structure it is easier to change in 1 place as opposed to 4 locations.